quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/KeycloakMain.java

```diff
@@ -17,10 +17,9 @@
 
 package org.keycloak.quarkus.runtime;
 
-import static org.keycloak.quarkus.runtime.Environment.isDevMode;
+import static org.keycloak.quarkus.runtime.Environment.*;
```

Comment:
We should avoid wildcards.

Reply:
oops. Something my IDE did, have to remove that automatic change to wildcard. will change.

> @DGuhr What do you think about this behavior:
> 
> ```
> kc.sh build --db postgres --db-username keycloak --db-password password --profile dev
> ```
> 
> The output is:
> 
> ```
> Updating the configuration and installing your custom providers, if any. Please wait.
> 2021-11-15 09:30:12,733 INFO  [io.qua.dep.QuarkusAugmentor] (main) Quarkus augmentation completed in 4205ms
> ```
> 
> Note that we are not showing the message saying that build was OK and "use show-config".
> 
> Shall we block users from using `build` command too when using `dev` configuration?

Hum, you are right, actually building with profile dev makes no sense anymore now, bc. using `start`would result in an error. We could instead add a hint `please use start-dev instead` or sth imo.